src/main/scala/memeid.scala

```diff
@@ -5,18 +5,30 @@ import cats.kernel._
 import java.lang.Long
 import java.util.{UUID => JUUID}
 
-case class UUID(msb: Long, lsb: Long) 
+case class UUID(juuid: JUUID) {
```

Comment:
```suggestion
case class UUID(private juuid: JUUID) {
```

Reply:
Good point, I'll commit it myself since it should be `private val juuid`.